pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -158,6 +158,7 @@ def __init__(
         provider: Optional[str] = None,
         machine_debug: bool = False,
         api_handler: QuantinuumAPI = DEFAULT_API_HANDLER,
+        **kwargs: QuumKwargTypes,
```

Comment:
Is this a good name for the variable?

Reply:
It's just copied from the `process_circuits()` signature.